src/CreatePackage.php

```diff
@@ -0,0 +1,226 @@
+                $repo
+            ));
+
+            return;
+        }
+
+        if (! mkdir($repo, 0775) && ! is_dir($repo)) {
+            $errOutput->writeln(sprintf(
+                '<error>Cannot create package directory "%s"</error>',
+                $repo
+            ));
+
+            return;
+        }
+
+        $namespace = $this->determineNamespace($input, $output, $repo);
+
+        $replacement = [
+            '{org}' => $this->getOrg($repo),
+            '{repo}' => $repo,
+            '{category}' => $this->getCategory($repo),
+            '{namespace}' => $namespace,
+            '{namespace-test}' => preg_replace('/^(Zend|ZF)/', '\\1Test', $namespace),
+            '{year}' => date('Y'),
+        ];
+
+        $this->copyFiles(__DIR__ . '/../template/', $repo . '/', $replacement);
+
+        $helper = $this->getHelper('question');
+        $question = new ConfirmationQuestion(
+            'Would you like to run composer and create initial commit in the new package? [<info>Y/n</info>] '
+        );
+
+        if ($helper->ask($input, $output, $question)) {
+            exec(sprintf(
+                'cd %s && composer update && git init . && git add . && git commit -am "Initial creation"',
+                $repo
+            ));
+        }
+    }
+
+    private function determineNamespace(InputInterface $input, OutputInterface $output, $repo)
+    {
+        $exp = explode('-', $repo);
+
+        $namespace = array_shift($exp) === 'zend' ? 'Zend' : 'ZF';
+
+        if ($exp[0] === 'expressive') {
+            $namespace .= '\Expressive';
+            array_shift($exp);
+        }
+
+        if (count($exp) === 1) {
+            $namespace .= '\\' . ucfirst(array_shift($exp));
+        }
+
+        $namespaces = [];
+        if ($exp) {
+            $base = ucwords(implode(' ', $exp));
+            $namespaces[] = $namespace . '\\' . str_replace(' ', '\\', $base);
+            $namespaces[] = $namespace . '\\' . str_replace(' ', '', $base);
+        } else {
+            $namespaces[] = $namespace;
+        }
+        $namespaces['c'] = 'custom';
+
+        $helper = $this->getHelper('question');
+
+        $question = new ChoiceQuestion(
+            'What namespace would you like to use?',
+            $namespaces,
+            0
+        );
+        $question->setErrorMessage('Invalid choice "%s". Please try again.');
+
+        $answer = $helper->ask($input, $output, $question);
+
+        if ($answer !== 'c') {
+            return $answer;
+        }
+
+        $question = new Question('Custom namespace: ');
+        $question->setValidator(function ($value) use ($repo) {
+            $firstPart = strpos($repo, 'zf') === 0 ? 'ZF' : 'Zend';
+            if (! preg_match('/^' . $firstPart . '(\\[A-Z][a-z0-9]*)+$/', $value)) {
+                throw new RuntimeException(sprintf(
+                    'Invalid namespace provided: %s',
+                    $value
+                ));
+            }
```

Comment:
This condition could either go away, or be modified to only run if the first segment is `zend` or `zf`.